Dockerfile

```diff
@@ -3,14 +3,20 @@ MAINTAINER Petr Sloup <petr.sloup@klokantech.com>
 
 RUN apt-get -qq update \
 && DEBIAN_FRONTEND=noninteractive apt-get -y install \
+    apt-transport-https \
     curl \
     unzip \
     build-essential \
     python \
     libcairo2-dev \
+    nodejs \
```

Comment:
You're right, that accidentally got left in after testing which version of nodejs gets installed into stretch. 7f06f09 removes it.